File: ocmsdk/src/main/java/com/gigigo/orchextra/core/data/dto/menus/ApiMenuContent.java
package com.gigigo.orchextra.core.data.dto.menus;


import com.gigigo.orchextra.core.data.dto.elements.ApiElement;
import com.google.gson.annotations.SerializedName;
import java.util.List;

public class ApiMenuContent {

    @SerializedName("_id")
    private String id;

    @SerializedName("slug")
    private String slug;

    @SerializedName("elements")
    private List<ApiElement> elements;

    public String getId() {
        return id;
    }

    public String getSlug() {
        return slug;
    }

    public List<ApiElement> getElements() {
        return elements;
    }
}

File: ocmsdk/src/main/java/com/gigigo/orchextra/core/data/api/dto/menus/ApiMenuContent.kt
package com.gigigo.orchextra.core.data.api.dto.menus

import com.gigigo.orchextra.core.data.api.dto.elements.ApiElement
import com.google.gson.annotations.SerializedName

class ApiMenuContent(
    @SerializedName("_id") val id: String?,
    @SerializedName("slug") val slug: String,
    @SerializedName("elements") val elements: List<ApiElement>?)